manifests/install.pp

```diff
@@ -39,12 +39,6 @@
       ensure  => $foreman::version,
       require => $repo,
     }
-    if $foreman::ipa_authentication and $foreman::configure_ipa_repo and $foreman::osreleasemajor == '6' {
-      package { 'mod_lookup_identity-selinux':
```

Comment:
And more specifically, it was only provided in the custom repo when the boolean was not yet available in the core selinux-policy-targeted.